packages/grafana-data/src/valueFormats/valueFormats.ts

```diff
@@ -257,8 +257,23 @@ export function getValueFormat(id?: string | null): ValueFormatter {
         return simpleCountUnit(sub);
       }
 
+      // Supported formats:
+      // currency:$           -> scaled currency ($1.2K)
+      // currency:financial:$ -> full currency ($1,234)
+      // currency:financial:€:suffix -> full currency with suffix (1,234€)
       if (key === 'currency') {
-        return currency(sub);
+        const keySplit = sub.split(':');
+
+        if (keySplit[0] === 'financial' && keySplit.length >= 2) {
+          const symbol = keySplit[1];
+          if (!symbol) {
+            return toFixedUnit(''); // fallback for empty symbol
+          }
+          const asSuffix = keySplit[2] === 'suffix' || keySplit[2] === 'true';
```

Comment:
Not sure I get the `keySplit[2] === 'true'` part? Like if the string is `currency:financial:$:true` we want that to behave the same as `currency:financial:$:suffix`?

Reply:
exactly! Imo it doesn't hurt to have redundancy here

Reply:
Should it be added to the docs? 🤔

Reply:
a very good point, I think it should, however, I don't think we follow this pattern anywhere 🤔 

Let me research, if is not a pattern I will remove this, I don't want to make it more confusing for users